sources/packages/backend/apps/db-migrations/src/sql/Student/Update-gender-options.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE
+  sims.students
+ALTER COLUMN
+  gender TYPE varchar(50);
+
+UPDATE
+  sims.students
+SET
+  gender = 'man'
+WHERE
+  gender = 'male';
+
+UPDATE
+  sims.students
+SET
+  gender = 'nonBinary'
+WHERE
+  gender = 'X';
```

Comment:
The X does not belong to the current list of values in the dropdown.
`nonBinary` would be equivalent to `diverse` right? Why was the `X` added?